capstone/src/test/java/com/google/sps/objects/TimeTest.java

```diff
@@ -17,7 +17,7 @@
 public class TimeTest {
 
   @Spy private Time mockedTime;
-  private static long EXPECTED_DATE = 1595894399000L;
+  private static long EXPECTED_DATE = 1595908799000L;
```

Comment:
Does this need to change or can you keep it to the old value? This keeps changing in PRs and it's possibly because y'all have different versions of this that keep getting uploaded

Reply:
I think this has something to do with whether this project is run on Cloud Shell or my personal computer. It always switches between two different values, where Cloud Shell is always one number and my personal computer is another value